### PR TITLE
Add option for tabindex attribute

### PR DESCRIPTION
**Pull request by huw**
This was useful for a personal project, inspired by [this css-trick](https://css-tricks.com/expanding-images-html5/). If you don't think it's appropriate for the master branch here, I'm happy to close this request 🙂.

### CODE REVIEW

**Comment by arve0** [comment]
I like it. :+1: Gives something extra that are less elegant with [attrs](https://www.npmjs.com/package/markdown-it-attrs).

File: index.js [hunk 1]
```diff
@@ -3,6 +3,8 @@
 module.exports = function implicitFiguresPlugin(md, options) {
   options = options || {};
 
+  var tabIndex = 1;
```
Comment (arve0) [inline index.js]: This should be inside `function implicitFigures`, to have it reset upon each `md.render()`.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "markdown-it-implicit-figures",
-  "version": "0.3.2",
+  "version": "0.4.0",
```
Comment (arve0) [inline package.json]: I'll bump version on publishing, please remove this.

File: test.js [hunk 3]
```diff
@@ -40,6 +40,14 @@ describe('markdown-it-implicit-figures', function() {
     assert.equal(res, expected);
   });
 
+  it('should add incremental tabindex to figures when opts.tabindex is set', function () {
```
Comment (arve0) [inline test.js:43]: Add a test to check that `tabIndex` is reset between each `md.render()`.

**Review by arve0 — CHANGES_REQUESTED** [review]
Please take a look at the comments.

File: test.js (hunk 3 above)
Comment (huw) [reply test.js:43]: Let me know if you think the proposed test is sensible—I don't change the source or expected output, just render and compare twice.

Comment (arve0) [reply test.js:43]: :+1:

**Comment by arve0** [comment]
Thanks for this. I've merged locally. Released in `v0.5.0`.

**Pull request closed without merging**